### PR TITLE
Update `CODEWONERS` file

### PR DESCRIPTION
**Pull request by peagasilva**

### CODE REVIEW

**Comment by wetransferplatform** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Please provide a summary in the Pull Request description</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  Skipped running XCResults Summary as `shouldSkip` returned true.
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  View more details on <a href="https://app.bitrise.io/build/d1d0ba15-b944-45c4-b58b-cdb378f73ae5" target="_blank">Bitrise</a>
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  Skipped running SwiftLint as `shouldSkip` returned true.
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against cf298faa4f985c8ba29640f3b308f8fb788ec6fe
</p>

**Review by BasThomas — APPROVED** [review]

File: .github/workflows/CODEOWNERS
Comment (BasThomas) [inline .github/workflows/CODEOWNERS:1]: Is this change intended?

Comment (peagasilva) [reply .github/workflows/CODEOWNERS:1]: Not really - thanks for spotting it. I'll revert it.

Comment (peagasilva) [reply .github/workflows/CODEOWNERS:1]: Addressed in https://github.com/WeTransfer/Diagnostics/pull/158/commits/819ca11aa5ed358d573d4c05bc6a8b0f07233f9d.

**Review by kairadiagne — APPROVED** [review]

**Comment by wetransferplatform** [comment]
Congratulations! :tada: This was released as part of [Release 4.6.0](https://github.com/WeTransfer/Diagnostics/releases/tag/4.6.0) :rocket:<p align="right"><i>Generated by <a href="https://github.com/WeTransfer/GitBuddy">GitBuddy</a></i></p>
